src/components/toast/toast.css

```diff
@@ -0,0 +1,12 @@
+.toast-msg {
+    color: #f5f5f5
+}
+
+.success-toast {
+    background-color: #407ebd !important;
```

Comment:
avoid using `!important`  . It will override user stylesheets

Reply:
I have imported toast from my component library, which comes with default color. Color was not working with specificity either. So, I used important tag there.